modules/web/src/app/node-data/basic/provider/anexia/component.ts

```diff
@@ -239,8 +327,8 @@ export class AnexiaBasicNodeDataComponent extends BaseFormValidator implements O
       anexiaNodeSpec: {
         [Controls.Cpus]: this.form.get(Controls.Cpus).value,
         [Controls.Memory]: this.form.get(Controls.Memory).value,
-        [Controls.DiskSize]: this.form.get(Controls.DiskSize).value,
-        [Controls.TemplateID]: this.form.get(Controls.TemplateID).value?.[AutocompleteControls.Main],
+        [Controls.Disks]: this.form.get(Controls.Disks).value,
```

Comment:
As mentioned in earlier comment, we need to send the `disks` property in the same format which we send for creating the cluster.